cpp/dolfinx/fem/FiniteElement.h

```diff
@@ -76,6 +76,14 @@ class FiniteElement
       const Eigen::Ref<const Eigen::Array<
           double, Eigen::Dynamic, Eigen::Dynamic, Eigen::RowMajor>>& X) const;
 
+  /// Evaluate all basis function derivatives of order order at given point in
```

Comment:
of order order sounds a bit strange, how about "Evaluate all basis function derivatives of given order at given point in the reference cell"

Reply:
:+1: